postcss.config.js

```diff
@@ -1,3 +1,42 @@
+const platformPostcssConfig = require('@hashicorp/platform-postcss-config')
+
+/**
+ * TODO: this is a spike to show that this configuration may resolve
+ * the issues we're having in dev-dot when using @custom-media.
+ * We likely want to incorporate this into `@hashicorp/platform-postcss-config`.
+ * Eg, maybe it should accept an optional "options" object, where we can
+ * pass additional files to `importFrom`?
+ */
+function alsoImportDevDotCustomMedia(postcssConfig) {
+  const newPlugins = postcssConfig.plugins.map((p) => {
```

Comment:
nit, naming for clarity:
```suggestion
  const newPlugins = postcssConfig.plugins.map((plugin) => {
```

Reply:
✅ Have updated this throughout the function body - I dig the clarity 👍